src/components/Answer/Answer.js

```diff
@@ -1,11 +1,22 @@
 import Question from '../../model/question';
 
 export default class Answer {
-  timeOfAnswer;
-  Question;
-  lifelineUsed = false;
+  constructor(timeOfAnswer, Question, answer, lifelineUsed) {
+    this.timeOfAnswer = timeOfAnswer;
+    this.Question = Question;
+    this.lifelineUsed = lifelineUsed;
+    this.answer = answer;
+  }
+
+  corectnessCheck() {
+    if (this.answer === this.Question.correct.toUpperCase()) {
```

Comment:
the answer is expected to be passed as UpperCase, but it can cause problems if it changes in the rendering of the quiz view. It will be safer to do the comparison of this.answer.toUpperCase()